frontend/src/components/InfoPage.jsx

```diff
@@ -6,11 +6,11 @@ function InfoPage() {
 
   return (
     <section className="main-section" style={{ backgroundImage: `url(${backgroundImage})` }}>
```

Comment:
Think this should be a main tag got each page component

Reply:
done